### PR TITLE
Pulled from master and add fetchSubjects effect

### PR DESCRIPTION
**Pull request by jotnguyen**
This is the final PR I am opening. Master changes were pulled and fetchSubjects effects were added

### CODE REVIEW

File: src/effects/browse.js [hunk 1]
```diff
@@ -4,9 +4,11 @@ import { CURRENT_TERM } from 'util/constants';
 const currentTermDoc = db.collection('terms').doc(CURRENT_TERM);
 
 /* eslint-disable import/prefer-default-export */
-export function fetchSchools() {
```
Comment (Joonpark13) [inline src/effects/browse.js]: We still need the `fetchSchools` effect.

**Review by Joonpark13 — CHANGES_REQUESTED** [review]

File: src/effects/browse.js (hunk 1 above)
Comment (jotnguyen) [reply src/effects/browse.js]: Oops I accidentally deleted that

**Review by Joonpark13 — APPROVED** [review]
You can also get rid of the `eslint-disable import/prefer-default-export` now that there is more than one effect here.

**Comment by Joonpark13** [comment]
[Implement fetchSubjects effect](https://trello.com/c/KbW5Tgzv/73-implement-fetchsubjects-effect)

**Pull request merged**